### PR TITLE
Refact/simplify ppx deriving tag cvclib

### PR DESCRIPTION
**Pull request by chrismamo1**
This PR simplifies the `_tags` file and fixes a few warnings triggered by the CVC code as mentioned in #13

### CODE REVIEW

File: src/cvc.ml [hunk 1]
```diff
@@ -295,9 +262,10 @@ struct
         | Ok (`RSA (n, e)) ->
             Ok {n; e}
         | Ok (`ECDSA _)
-        | Ok (`RSA _)
         | Ok `UNKNOWN ->
             Error "CVC: Algorithm OID and parameters do not match."
+        | Error _ as err -> (* aliasing to avoid unnecessary allocation *)
```
Comment (emillon) [inline src/cvc.ml]: Can you remove these comments? this pattern is a well-known one.

Comment (chrismamo1) [reply src/cvc.ml]: Removed.

**Comment by emillon** [comment]
Can you rebase your branch on top of master? thanks

**Comment by chrismamo1** [comment]
Rebased and pushed.

**Comment by emillon** [comment]
Thanks, merging this.

**Pull request merged**